docs/source/installation.rst

```diff
@@ -177,25 +177,26 @@ In the ``anaconda prompt`` terminal:
 
 5. Activate the environment: :code:`conda activate sktime-dev`
 
-6. Build an editable version of sktime :code:`pip install -e .[dev]`
-In order to install all soft dependencies, use :code:`pip install -e .[dev]`
-If you are on Mac, use :code:`pip install -e ".[dev]"`
+6. Build an editable version of sktime.
+In order to install only the dev dependencies, :code:`pip install -e .[dev]`
 If you also want to install soft dependencies, install them individually, after the above,
 or instead use: :code:`pip install -e .[all_extras,dev]` to install all of them.
 
-7. If everything has worked you should see message "successfully installed sktime"
+    .. note::
 
-Some users have experienced issues when installing NumPy, particularly version 1.19.4.
+        If this step results in a "no matches found" error, it may be due to how your shell handles special characters.
 
-.. note::
+        - Possible solution: use quotation marks:
+
+            .. code-block:: bash
 
-    If step 4. results in a "no matches found" error, it may be due to how your shell handles special characters.
+                pip install -e ."[dev]"
 
-    - Possible solution: use quotation marks:
+7. If everything has worked you should see message "successfully installed sktime"
+
+Some users have experienced issues when installing NumPy, particularly version 1.19.4.
```

Comment:
Non-blocking suggestion: This can probably be removed, as `sktime` now needs `numpy>1.21`.

https://github.com/sktime/sktime/blob/b1921da2d613a900b9a9f6e0ce1319fe27b46f1b/pyproject.toml#L48

Reply:
hm, didn't notice! Interesting, when/why did we introduce that lower bound?

Reply:
As per blame, it was done in #1921. Way before I was even aware of `sktime`, no idea.

Reply:
ah, I asked the same question in #1921. It was said there were security risks below that bound, so I just took that on face value without checking. Probably true.

Reply:
Hi @julia-kraus, will you remove this line then? it's outdated and no more required. Everything else is in this PR looks good, and it can be merged then.

(not strictly related to the PR title, but probably it isn't worth a new PR just for one line removal.)